Dockerfile.rhel7

```diff
@@ -1,11 +1,11 @@
-FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.14 AS builder
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-openshift-4.6 AS builder
```

Comment:
Hmm, should this be `Dockerdile.rhel8` or just `Dockerfile.rhel` now?

Reply:
https://github.com/openshift/ocp-build-data/blob/openshift-4.6-rhel-8/images/cluster-version-operator.yml#L7 is the Dockerfile in use via ocp-build-data

Reply:
I've filed #444 hung on this same bug using a symlink from a generic `Dockerfile.rhel`.  If/when that lands, we can pivot ocp-build-data.